elephant/utils.py

```diff
@@ -288,18 +298,18 @@ def round_binning_errors(values, tolerance=1e-8):
     if isinstance(values, np.ndarray):
         num_corrections = correction_mask.sum()
         if num_corrections > 0:
-            warnings.warn(f'Correcting {num_corrections} rounding errors by '
-                          f'shifting the affected spikes into the following '
-                          f'bin. You can set tolerance=None to disable this '
-                          'behaviour.')
+            logger.info(f'Correcting {num_corrections} rounding errors by '
+                         'shifting the affected spikes into the following '
+                         'bin. You can set tolerance=None to disable this '
+                         'behaviour.')
```

Comment:
Maybe better a warning, not info...
```suggestion
            logger.warning(f'Correcting {num_corrections} rounding errors by '
                         'shifting the affected spikes into the following '
                         'bin. You can set tolerance=None to disable this '
                         'behaviour.')
```